### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by TiagoMarinho** [AtomLinter/linter-htmlhint#24]
I ran through an exception while closing a HTML tab and Atom suggested me to create this issue, so that's what I'm doing.

1. Close a HTML tab
2. Uncaught Exception is triggered

**Atom Version**: 0.199.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.10.3
**Thrown From**: [linter-htmlhint](https://github.com/AtomLinter/linter-htmlhint) package, v0.0.15

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/Tiago/.atom/packages/linter-htmlhint/lib/linter-htmlhint.coffee:47

TypeError: undefined is not a function
  at LinterHtmlhint.destroy (/Users/Tiago/.atom/packages/linter-htmlhint/lib/linter-htmlhint.coffee:47:17)
  at LinterView.remove (/Users/Tiago/.atom/packages/linter/lib/linter-view.coffee:252:7)
  at /Users/Tiago/.atom/packages/linter/lib/linter-view.coffee:124:8
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TextEditor.module.exports.TextEditor.destroyed (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:265:27)
  at TextEditor.module.exports.Model.destroy (/Applications/Atom.app/Contents/Resources/app.asar/src/model.js:45:58)
  at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:444:18)
  at HTMLDivElement.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:183:22)
  at space-pen-ul.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at space-pen-ul.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```

### Commands

```
     -2:39.3.0 core:move-right (atom-text-editor.editor.mini.is-focused)
     -2:38.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:34 core:move-right (atom-text-editor.editor.mini.is-focused)
     -2:32.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:08.1.0 core:backspace (atom-text-editor.editor.is-focused)
     -2:07.3.0 core:move-down (atom-text-editor.editor.is-focused)
     -2:07.0 core:backspace (atom-text-editor.editor.is-focused)
     -2:00.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:40.5.0 core:paste (atom-text-editor.editor.is-focused)
  6x -1:31 core:backspace (atom-text-editor.editor.is-focused)
     -1:03.9.0 core:move-left (atom-text-editor.editor.is-focused)
     -1:03.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:03.1.0 core:move-right (atom-text-editor.editor.is-focused)
     -0:59.9.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:58.1.0 editor:newline (atom-text-editor.editor.is-focused)
  3x -0:37.6.0 core:backspace (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "wrap-guide"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-ternjs, v0.5.25
autocomplete-paths, v1.0.2
color-picker, v1.7.0
language-js-specs, v0.1.0
linter, v0.12.2
linter-htmlhint, v0.0.15
linter-jshint, v0.1.4
minimap, v4.8.0
minimap-bookmarks, v0.1.0
minimap-find-and-replace, v4.2.0
minimap-git-diff, v4.1.2
minimap-highlight-selected, v4.2.3
minimap-selection, v4.2.0

# Dev
No dev packages
```

**Comment by kevinsawicki** [comment]
This was fixed by #22 and published in 0.0.16
